packages/hooking/manager.ts

```diff
@@ -181,3 +181,13 @@ export class HookManager {
 }
 
 export const hookManager = new HookManager();
+
+// convenience function to register hooks
+export function registerReplaceHook(
```

Comment:
Should we add convenience functions for `Before` and `After` as well?

Reply:
Added!